server/controllers/usersController.js

```diff
@@ -12,8 +11,8 @@ module.exports.insertPDF = (req, res) => {
 
 module.exports.getPDF = (req, res) => {
   const { fileUse } = req.params;
-  const userId = req.user.id;
+  const userId = req.user.id || 1;
```

Comment:
Defaulting to 1 here is not preferable. Instead we should kick back an unauthorized error. See the save jobs or applied jobs controllers.

Reply:
Example snippet in Slack now

Reply:
Just error fixing on my end to proceed with other changes. Feel free to handle however